lib/index.js

```diff
@@ -19,6 +19,19 @@ $(document).ready(() => {
   })
 })
 
+const filterFoods = () => {
+  var filter = $('#food-filter-input').val().toUpperCase()
+  var foodNameNodes = $('.food-item-name')
+  for(var i = 0; i < foodNameNodes.length; i++) {
+    var name = $(foodNameNodes[i])
+    if (name.text().toUpperCase().indexOf(filter) > -1) {
```

Comment:
possibly use `!== -1` It feels more explicit to me since the results of  indexOf are only -1 on no matching substring.